### PR TITLE
Dtd 599

### PR DESCRIPTION
**Pull request by Jxr227**

### CODE REVIEW

**Review by vivekhmrc — APPROVED** [review]

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala [hunk 1]
```diff
@@ -40,6 +43,254 @@ class TimeToPayProxySteps extends ScalaDsl with EN with Eventually with Matchers
     TimeToPayProxyRequests.createPlanRequestBody(dataTable)
   }
 
+  Given("a create plan request with an empty customer reference") { () =>
```
Comment (rogerball) [inline src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala]: Is this already covered by 'Scenario TTP Promote an Arrangement request to create a plan for storage -empty customerReference'?

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala [hunk 2]
```diff
@@ -40,6 +43,254 @@ class TimeToPayProxySteps extends ScalaDsl with EN with Eventually with Matchers
...
+      ),
+      Seq(DebtItemCharge("Debt1", "1525", "1000", 500, LocalDate.of(2019, 10, 5), Some(Seq(ChargePayment(LocalDate.now().minusWeeks(5), 100))))),
+      Seq(Payment("BACS", "PRef1")),
+      Seq(PostCode("NW9 5XW", LocalDate.now().minusMonths(6))),
+      Seq(Instalment("Debt1", LocalDate.now().plusMonths(1), 500, 500, 0.26, 1, 50, 10000 - 500))
+    )
+    ScenarioContext.set("createPlanRequest", Json.toJson(request).toString())
+  }
+
+  Given("a create plan request with an empty channel identifier") { () =>
```
Comment (rogerball) [inline src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala]: Already covered by 'Scenario: TTP Promote an Arrangement request to create a plan for storage -empty channelIdentifier'?

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala (hunk 1 above)
Comment (Jxr227) [reply src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala]: Yeah - this step is to provide data for that scenario

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala (hunk 2 above)
Comment (Jxr227) [reply src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala]: Yeah - this step is to provide data for that scenario

**Review by thadeusssewanyana — APPROVED** [review]

**Pull request merged**